Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM node:16.14-alpine
-COPY . /usr/cutiebot
+FROM node:lts-alpine
```

Comment:
would setting v16 explicitly be better, as when v18 becomes lts (2022-10-25) our app may stop building successfully